src/System.Private.CoreLib/shared/System/Collections/Generic/List.cs

```diff
@@ -1023,7 +1024,7 @@ public void Sort(Comparison<T> comparison)
 
             if (_size > 1)
             {
-                ArraySortHelper<T>.Sort(_items, 0, _size, comparison);
+                ArraySortHelper<T>.Sort(_items, 0, _size, comparison!);
```

Comment:
// TODO-NULLABLE: https://github.com/dotnet/csharplang/issues/538 ? To fix when we can annotate ThrowHelpers.